src/styles.scss

```diff
@@ -215,3 +215,53 @@ mat-card.dashboard-widget {
 .mat-select-disabled .mat-select-value {
   color: black;
 }
+
+/* Replace the max java integer in the page size dropdown with the label 'All'  */
+mat-option[ng-reflect-value='2147483647'] > span {
+	display: none;
+}
+
+/* Replace the max java integer in the page size dropdown with the label 'All'  */
+mat-option[ng-reflect-value='2147483647']:last-child:after {
```

Comment:
Is this still relevant? I dont see the max integer being used

Reply:
Right; I've deleted that.